core/src/commonMain/kotlin/com/ustadmobile/core/domain/passkey/GetCredentialUseCase.kt

```diff
@@ -0,0 +1,6 @@
+package com.ustadmobile.core.domain.passkey
+
+interface GetCredentialUseCase {
```

Comment:
It would be better for this usecase to follow the same pattern as other use cases which are related to a given learning space (e.g. enrollment, invite, etc) - let's use the DI scope with LearningSpace as a parameter, not use domain as a parameter to the invoke function.

e.g.


```interface GetCredentialUseCase

suspend operator fun invoke(): CredentialResult
```

Then in the implementation can use something like this:
```
class GetCredentialUseCaseImpl(
   private val learningSpace: LearningSpace
) {
    private val domain: String by lazy {
       Url(learningSpace.url).host
    }
     
   
    operator fun invoke() {
       //logic here    
    }  
}

```